pwa/src/components/applications/applicationsTable.tsx

```diff
@@ -74,7 +100,7 @@ export default function ApplicationsTable() {
                       headerName: " ",
                       renderCell: (item: { id: string }) => {
                         return (
-                          <Link className="utrecht-link d-flex justify-content-end" to={`/applications/${item.id}`}>
+                          <Link to={`/applications/${item.id}`}>
```

Comment:
Why is this change necessary? 

There's at least one thing breaking due to this change, the _Edit_ button was aligned right in its table with the classes `d-flex justify-content-end`.